src/patternfly/components/Button/examples/button-types-example.hbs

```diff
@@ -1,19 +1,22 @@
 {{#> button button--modifier="pf-m-primary"}}
-    Primary
+  Primary
 {{/button}}
 {{#> button button--modifier="pf-m-secondary"}}
-    Secondary
+  Secondary
 {{/button}}
 {{#> button button--modifier="pf-m-tertiary"}}
-    Tertiary
+  Tertiary
 {{/button}}
 {{#> button button--modifier="pf-m-danger"}}
-    Danger
+  Danger
 {{/button}}
 {{#> button button--modifier="pf-m-link"}}
-    {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
-    Link Button
+  {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
+  Link Button
 {{/button}}
 {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Remove"'}}
-    {{> button-icon button-icon--type="times"}}
+  {{> button-icon button-icon--type="times"}}
+{{/button}}
+{{#> button button--modifier="pf-m-inline pf-m-link"}}
```

Comment:
I wonder if this should be called "Inline Link Button"

Reply:
updated.